css/index.css

```diff
@@ -104,7 +123,117 @@ p {
 .middle-content .boxes .box {
     width: 12.5%;
     height: 100px;
-    background: black;
+    background: blueviolet;
+    margin: 20px 2.5%;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.box1 {
```

Comment:
Interesting to see you chose to modify the HTML instead of using the `:first-child`, `:nth-child(child number)`, `:last-child` psuedo selectors. Many ways to skin that cat that is definitely one of them.